.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,45 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - main
+      - 8.0
+      - 8.1
+      - 7.17
```

A more generic approach? like https://github.com/elastic/beats/blob/0def363871565f6024f034965fa2a26fa769e51e/.github/workflows/macos-auditbeat.yml#L8-L11

```suggestion
    branches:
      - main
      - 8.*
      - 7.17
```

---

Yes +1 on this, I am still new to action so every comment are welcome. I will make the change.